src/app/client/src/app/modules/questionnaire/questionnaire.service.ts

```diff
@@ -35,10 +42,10 @@ export class QuestionnaireService {
     let sections = evidence.sections;
     let answers = this.getSectionData(sections, formValues);
     let payloadData = {
-      externalid: "OB", //todo
+      externalId: evidence.externalId,
       answers: answers,
-      // startTime: 1620977187792, //todo
-      // endTime: 1620977187792, //todo
+      startTime: evidence.startTime,
+      endTime: Date.now(),
```

Comment:
should we do this here?  Cant we do in init of respective components?

Reply:
this. is evidence start time (which we set when we call the get details api) and end time (which we set before making make api) so i am doing in service.